src/ClpCInterface.jl

```diff
@@ -245,12 +246,14 @@ mutable struct ClpSolve
     function ClpSolve()
         p = @clpsolve_ccall new Ptr{Cvoid} ()
         prob = new(p)
+        println("Creating ClpSolve: $(p)")
         finalizer(delete_solve, prob)
         return prob
     end
 end
 
 function delete_solve(solve::ClpSolve)
+    println("Finalizing ClpSolve: $(solve.p)")
```

Comment:
The docs of `finalizer` says
```julia
finalizer(f, x)

  Register a function f(x) to be called when there are no program-accessible references to x, and return x. The type of x must be a mutable struct, otherwise the behavior of this function is unpredictable.

  f must not cause a task switch, which excludes most I/O operations such as println. @schedule println("message") or ccall(:jl_, Void, (Any,), "message") may be helpful for debugging purposes.
```
Might the `println` risk creating further errors?

Reply:
No, that by itself is not the issue. If there's a race that leads to a double free we need to fix it.